cabal.project

```diff
@@ -48,7 +48,9 @@ package cryptonite
   flags: -support_rdrand
 
 package snap-server
-  flags: +openssl
+  -- OpenSSL support requires HsOpenSSL which will cause problems in the near future
+  -- https://github.com/haskell-cryptography/HsOpenSSL/issues/95
+  flags: -openssl
```

Comment:
Unless `snap-server` remains somewhere in the node's dependency graph, this flag from now only affects `cardano-tracer`. That, however, will need to account for a non-default, non-SSL build of its dependency within the application code.

Merging this flag change would break `cardano-tracer` currently, please remove and postpone.

Reply:
This currently builds with this change.

The `-openssl` flag on enables SSL for `cardano-tracer` when `cardano-tracer` is built with the `-rtview` flag which is not one be default.

Reply:
Flags `-openssl` and `+rtview` result in a broken service (i.e., unable to start up). Handling this case needs to be adjusted in the application code before any merge.